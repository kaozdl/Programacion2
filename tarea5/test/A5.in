# A
ingresar_localidades (21,goya) (15,ulsan) (70,tambo) .
iniciar_pedidos 5
ingresar_pedidos ulsan (1000,chapas) 4 tambo (500,ropa) 5 ulsan (2000,comestibles) 3 .
imprimir_totales
procesar_rango 2
procesar_rango 3
procesar_rango 4
imprimir_totales
Fin
