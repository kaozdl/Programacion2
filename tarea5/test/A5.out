1>2>3>
 IDENTIFICADORES DE LOCALIDADES 
(1,goya)(2,tambo)(3,ulsan)
4>5>INFORME POR RANGOS
3000
0
0
500
0
6>goya

7>8>tambo
(500,ropa)
9>INFORME POR RANGOS
3000
0
0
0
0
10>Fin.
